packages/minimongo/selector.js

```diff
@@ -649,6 +690,108 @@ ELEMENT_OPERATORS = {
+
+        return typeof _.find(operand, function (op, i) {
+            var bitsSetOp = get8BitsSet(op);
+            var bitsSetVal = get8BitsSet(value[i]);
+
+            return typeof _.find(bitsSetOp, function (bit) {
+                return bitsSetVal.indexOf(bit) === -1;
+              }) === 'undefined';
+          }) === 'undefined';
+      };
+    }
+  },
+  $bitsAllClear: {
+    compileElementSelector: function (operand, valueSelector, matcher) {
+      if (!isArray(operand) && !(operand instanceof Uint8Array))
+        throw Error("$bitsAllClear has to be an Array");
+      return function (value) {
+        if (typeof value !== 'number' && !(value instanceof Uint8Array))
+          return false;
+
+        if (value === 0)
+          return true;
+
+        operand = ensureOperandUint8Array(operand);
+
+        if (!(value instanceof Uint8Array))
+          value = convertNumberToUint8Array(value);
+
+        return _.filter(operand, function (op, i) {
+          var bitsSetOp = get8BitsSet(op);
+          var bitsSetVal = get8BitsSet(value[i]);
+
+          return typeof _.find(bitsSetOp, function (bit) {
+            return bitsSetVal.indexOf(bit) !== -1;
+          }) !== 'undefined';
+        }).length === 0;
+      };
+    }
+  },
+  $bitsAllSet: {
+    compileElementSelector: function (operand, valueSelector, matcher) {
+      if (!isArray(operand) && !(operand instanceof Uint8Array))
+        throw Error("$bitsAllSet has to be an Array");
+      return function (value) {
+        if (typeof value !== 'number' && !(value instanceof Uint8Array))
+          return false;
+
+        operand = ensureOperandUint8Array(operand);
+
+        if (!(value instanceof Uint8Array))
+          value = convertNumberToUint8Array(value);
+
+        return _.filter(operand, function (op, i) {
+            var bitsSetOp = get8BitsSet(op);
+            var bitsSetVal = get8BitsSet(value[i]);
+
+            return typeof _.find(bitsSetOp, function (bit) {
+                return bitsSetVal.indexOf(bit) === -1;
+              }) !== 'undefined';
+          }).length === 0;
+      };
+    }
+  },
+  $bitsAnySet: {
+    compileElementSelector: function (operand, valueSelector, matcher) {
+      if (!isArray(operand) && !(operand instanceof Uint8Array))
+        throw Error("$bitsAnySet has to be an Array");
+      return function (value) {
+        if (typeof value !== 'number' && !(value instanceof Uint8Array))
+          return false;
+
+        operand = ensureOperandUint8Array(operand);
+
+        if (!(value instanceof Uint8Array))
+          value = convertNumberToUint8Array(value);
+
+        return typeof _.find(operand, function (op, i) {
+            var bitsSetOp = get8BitsSet(op);
+            var bitsSetVal = get8BitsSet(value[i]);
+
+            return typeof _.find(bitsSetOp, function (bit) {
+                return bitsSetVal.indexOf(bit) !== -1;
+              }) === 'undefined';
+          }) === 'undefined';
+      };
+    }
+  },
```

Comment:
You should be able to use more modern ECMAScript in this file as `ecmascript` is included in its [`package.js`](https://github.com/davidworkman9/meteor/blob/40a535add85f7f793d60540b045b6260b22e9053/packages/minimongo/package.js#L19)